src/Collection.php

```diff
@@ -273,6 +277,35 @@ public function count($filter = [], array $options = [])
         return $operation->execute($server);
     }
 
+    /**
+     * Gets the number of documents matching the filter.
+     *
+     * @see Count::__construct() for supported options
+     * @param array|object $filter  Query by which to filter documents
+     * @param array        $options Command options
+     * @return integer
+     * @throws UnexpectedValueException if the command response was malformed
+     * @throws UnsupportedException if options are not supported by the selected server
+     * @throws InvalidArgumentException for parameter/option parsing errors
+     * @throws DriverRuntimeException for other driver errors (e.g. connection errors)
+     */
+    public function countDocuments($filter = [], array $options = [])
```

Comment:
As noted elsewhere, this default value on `$filter` is sufficient to provide a convenience to users. The Operation class can omit the default value.